src/main/java/org/opensearch/neuralsearch/processor/factory/RerankProcessorFactory.java

```diff
@@ -72,6 +72,7 @@ public SearchResponseProcessor create(
                 );
                 return new MLOpenSearchRerankProcessor(description, tag, ignoreFailure, modelId, contextFetchers, clientAccessor);
             case BY_FIELD:
+                boolean DEFAULT_REMOVE_TARGET_FIELD = false;
```

Comment:
it's a constant, please declare it on top of the class with other static variables.